requirements.in

```diff
@@ -13,4 +13,3 @@ flower
 # version, so pin it explicitly here.
 tornado<6
 memote>=0.9.11
-pytest<4.1
```

Comment:
`pytest` is a direct dependency of memote so it will be pulled in. I don't remember why it was pinned at <4.1 explicitly here, though, memote is fine with newer versions.